src/arena/CreateSprintTemplate.js

```diff
@@ -143,19 +146,108 @@ function CreateSprintTemplate(props) {
     getSanityItems();
   }, []);
 
+  useEffect(() => {
+    getConfiguration();
+  }, []);
+
+  useEffect(() => {
+    handleChange();
+  }, [title, columns, handleChange]);
+
+  // pulls the /templates api and sets the missions to the templates
```

Comment:
Nit: Doesn't need any action but it might be more readable to future contributors if instead of missions and setMissions, the variables were called something like "existingTemplates and setExistingTemplates" - definitely not worth worrying about to land this PR but I just wanted to mention it as I got myself a little turned around thinking about missions!

Reply:
Since Im already going to resubmit Ill change it, what you are saying makes perfect sense!